tests/test_other_scripts.py

```diff
@@ -61,20 +60,6 @@ def test_verify_data(self):
         assert report['missing_labels'] == 0
         assert report['missing_text'] == 0
 
-    def test_verify_bad_data(self):
```

Comment:
was this taking a long time?

Reply:
No. I was finding it nondeterministic.

Reply:
I think Paul was complaining about verify data breaking. Maybe that's what I was observing. 🤔

(Either way, I just decided to yeet this test, lol, maybe that's bad reasoning)

Reply:
that's kinda odd that a verify data script was nondeterministic? I'm curious what was breaking if that's what was going on - nevertheless fine with yeeting as it looks like we have tests to make sure verify data works on good data